✨ feat: add basic readme (#90)
velocitylabs-org/turtle: README.md

<a id="readme-top"></a>

<!-- PROJECT LOGO -->
<br />
<div align="center">
  <a href="https://turtle.cool">
    <img src="./app/public/turtle.svg" alt="Logo" width="80" height="80">
  </a>

  <h3 align="center">Turtle</h3>

  <p align="center">

>     Token transfers done right!

Reviewer: I like that (I wrote it somewhere originally I believe 😆) but we should use the official tagline:

```suggestion
    Frictionless cross-chain transfers
```

Reviewer 2: Maybe we should update this as well in the app? 
<img width="530" alt="image" src="https://github.com/user-attachments/assets/b0b36db9-2f73-44ae-85b0-bc6505ddd7cc">

Reviewer: for sure, let's do it in this same PR 👍

    <br />
    Made with 💚 by Velocity Labs
    <br/>
    <a href="https://app.turtle.cool"><strong> Visit the app »</strong></a>
    <br />
  </p>
</div>

<!-- ABOUT THE PROJECT -->

## About The Project

Turtle is your go to app when it comes to cross-chain transfers that leverage fully trustless infrastructure. We aim to provide a unified experience to transfer tokens anywhere.

Features:

> - Make transfers between blockchains in a decentralized and trustless way. With a mind blowing UI.
> - Powered by Snowbridge. Transfer ERC20 tokens between Ethereum and Polkadot.
> - Soon to be added: Asset Transfer API support. Transfer tokens seamlessly between all parachains.

Reviewer: ```suggestion
- Make transfers between blockchains in a decentralized and trustless way 
- Intuitive UI that helps you along the way
- Bridge ERC20 tokens between Ethereum and Polkadot -  Powered by Snowbridge 
- [Soon]: Seamless XCM Transfers between all Polkadot parachains
```

<br/>

> Turtle was developed with Typescript, Nextjs, and React.

Reviewer: ```suggestion
Turtle is developed with Typescript, Nextjs, and React.
```

<p align="right">(<a href="#readme-top">back to top</a>)</p>

<!-- CONTRIBUTING -->

## Contributing

> If you have a suggestion that would make turtle even better, please fork the repo and create a pull request. You can also simply open an issue.
> Don't forget to give the project a star! Thanks again!

Reviewer: ```suggestion
If you have a suggestion or want to contribute to Turtle, either fork the repo and create a pull request or simply open an issue.

Don't forget to give the project a star. Thanks for your support!
```

### Top contributors:

<a href="https://github.com/velocitylabs-org/turtle/graphs/contributors">
  <img src="https://contrib.rocks/image?repo=velocitylabs-org/turtle" alt="contrib.rocks image" />
</a>

<p align="right">(<a href="#readme-top">back to top</a>)</p>

<!-- ROADMAP -->

## Roadmap

- Coming Soon

<p align="right">(<a href="#readme-top">back to top</a>)</p>

<!-- LICENSE -->

## License

Distributed under the MIT License. See `LICENSE` for more information.

<p align="right">(<a href="#readme-top">back to top</a>)</p>

<!-- CONTACT -->

## Contact

> Nuno - nuno@velocitylabs.org

Reviewer: I don't mean to be creating hierarchies but it's important to leave this imo:

```suggestion
Nuno (product & tech lead) - nuno@velocitylabs.org
```

Author: 🫡 Sir yes Sir
